test-data/movsx.yaml

```diff
@@ -146,11 +137,9 @@ code:
 post:
   - r1.type=number
   - r1.svalue=[32768, 32770]
-  - r1.uvalue=[32768, 32770]
   - r1.svalue=r1.uvalue
```

Comment:
If you have "pc[1]=r0.svalue=r0.uvalue" elsewhere then why wouldn't this be something like
"r1.uvalue=r1.svalue=[32768, 32770]"?

Reply:
I though it was too much.